fk/teamfetch.go

```diff
@@ -61,28 +61,16 @@ func teamFetch(unattended bool) exitCode {
 }
 
 func doUpdateTeam(myTeam *team.Team, me *team.Person, unattended bool) (err error) {
-	alwaysDownload := !unattended
-
 	printHeader(myTeam.Name)
 
-	unlockedKey, err := getUnlockedKey(me.Fingerprint, unattended)
-	if err != nil {
-		out.Print(ui.FormatFailure(
-			"Failed to unlock key to check for team updates", []string{
-				"Checking for updates to the team requires an unlocked key",
-				"as the team roster is encrypted.",
-			}, err))
-		return err
-	}
-
 	var updatedTeam *team.Team
-	if updatedTeam, err = fetchAndUpdateRoster(*myTeam, unlockedKey, alwaysDownload); err != nil {
+	if updatedTeam, err = fetchAndUpdateRoster(*myTeam, *me, unattended); err != nil {
```

Comment:
In a previous commit, I remember `me` was a fingerprint. I wonder whether we should just be passing a `fingerprint` to the functions in this file, rather than a `Person`. Thoughts?